sdk/storage_v2.go

```diff
@@ -96,14 +96,14 @@ func (s *StorageV2Service) GetReplicators(ctx context.Context, rpOpts *Replicato
 	return rspDTO.toStruct(s.client.NetworkType())
 }
 
-func (s *StorageV2Service) GetDownloadChannelInfo(ctx context.Context, downloadChannelId *Hash) (*DownloadChannel, error) {
+func (s *StorageV2Service) GetDownloadChannelInfo(ctx context.Context, downloadChannelId *Hash) ([]*DownloadChannel, error) {
```

Comment:
why array?

Reply:
The function in the rest server expects an array. 
Is it possible that there can be 2 different entries having the same `DownloadChannelId`?

Reply:
It's not possible to have two same `DownloadChannelId`. So the route in REST should return `sendOne()`

Reply:
I've updated in REST (https://github.com/proximax-storage/js-xpx-chain-rest/pull/131/commits/56d6fee16d364db676f253b823b82758a6d9dd23) and reverted GetDownloadChannelInfo changes (7f87330)